packages/gatsby-source-shopify/src/gatsby-node.js

```diff
@@ -53,10 +53,22 @@ export const sourceNodes = async (
     verbose = true,
     paginationSize = 250,
     includeCollections = [SHOP, CONTENT],
+    shopifyQueries = {},
   }
 ) => {
   const client = createClient(shopName, accessToken, apiVersion)
 
+  const defaultQueries = {
+    articles: ARTICLES_QUERY,
+    blogs: BLOGS_QUERY,
+    collections: COLLECTIONS_QUERY,
+    products: PRODUCTS_QUERY,
+    shopPolicies: SHOP_POLICIES_QUERY,
+    pages: PAGES_QUERY,
+  }
```

Comment:
can this refactored to export the map directly from `./queries` like in the readme?

Reply:
--> #25811 `refactor(gatsby-source-shopify): inline defaultqueries`